git/week1/jsweek1.js

```diff
@@ -0,0 +1,86 @@
+// Homework javascript1-week1
+
+//  Age-ify (A future age calculator)
+
+var yearOfBirth = 1987;
+var yearFuture = 2030;
+var age = yearFuture - yearOfBirth;
+console.log('You will be ' + age + ' years old in ' + yearFuture);
```

Comment:
Check Template literals to log this sentence in a more elegant way https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals